import mysql.connector
from mysql.connector import MySQLConnection, Error
from sqlconnec import connect

def remove_cve():
    "Remove CVE from database"
    
    cve = input("Enter CVE-ID (Ex: CVE-2024-1234): ")

    conn = connect()
    cursor = conn.cursor()

    try:
        cursor.execute("DELETE FROM CVE_ID WHERE ID = %s", (cve,))
        conn.commit()
        print("CVE-ID removed")

    except Error as e:
        print("Error:", e)
        conn.rollback()
    finally:
        if conn.is_connected():
            cursor.close()
            conn.close()